web/modules/custom/feedback/feedback.install

```diff
@@ -1,55 +0,0 @@
-<?php
-
-# /**
-#  * @file
-#  * Installation functions for feedback module.
-#  */
-#
-# use Drupal\user\RoleInterface;
-#
-# /**
-#  * Implements hook_schema().
-#  */
-# function feedback_schema () {
```

Comment:
I don't think this is needed as well as if you define entity table in db will be created automatically. E.g. look at node module